src/sentry/static/sentry/app/views/projectInstall/platform.jsx

```diff
@@ -122,7 +123,13 @@ class ProjectInstallPlatform extends React.Component {
           ) : this.state.error ? (
             <LoadingError onRetry={this.fetchData} />
           ) : (
-            <DocumentationWrapper dangerouslySetInnerHTML={{__html: this.state.html}} />
+            <>
```

Comment:
We prefer the long-form for `Fragment` because it's easier to google for newcomers.

```suggestion
            <React.Fragment>
```

Reply:
kk, no problem )

Reply:
done